color_ranges:
  red:
    lower_bound:
      - 0
      - 10
      - 80
    upper_bound:
      - 50
      - 70
      - 200
